pep-0467.txt

```diff
@@ -73,8 +62,30 @@ second argument is the fill byte to use (defaults to ``\x00``)::
     >>> bytearray.fromsize(5, b'\x0a')
     bytearray(b'\x0a\x0a\x0a\x0a\x0a')
 
-``fromsize`` will behave just as the current constructors behave when passed a single
-integer, while allowing for non-zero fill values when needed.
+``fromsize`` will behave just as the current constructors behave when passed a
+single integer, while allowing for non-zero fill values when needed.
+
+
+Deprecation of current "zero-initialised sequence" behaviour without removal
```

Comment:
IMO that should not be called "deprecation". "Discouragement" sounds right though.

Reply:
I'm fine with "discouragement".

Reply:
Done, the word "deprecate" and variants thereof no longer appears in the PEP.